src/main/java/edu/uwaterloo/ece658/service/DataTransmissionService.java

```diff
@@ -15,19 +27,124 @@
 @Stateless
 public class DataTransmissionService {

```

Comment:
Is there any updating methods? For example a user wants to add new tags to a photo or remove a tag from a photo(this user must be the uploader of that photo. Otherwise, he/she doesn't have permission to do this operation).

Reply:
DataTransmissionService might not be a good name anymore. Currently you're updating the database if new photo is uploaded. Or when deleting a photo. If updating information methods are also added, we need a new name for this Service.

Reply:
I'll add the update methods